### PR TITLE
Input image

### PR DESCRIPTION
**Pull request by Sahil25061999**
created input component

### CODE REVIEW

File: component/input/input.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@import '/index.css';
+
+.textbox {
+  padding: 0.5rem 1rem;
```
Comment (gauravsinhaweb) [inline component/input/input.css]: give values in integer and use px if the value is less than 10px

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

File: component/input/input.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="input.css" />
+    <title>Input</title>
+  </head>
+  <body>
+    <h2>Input section</h2>
+    <section class="temp_container">
```
Comment (sanjitsarkar) [inline component/input/input.html]: What's temp_container.
Use proper and meaningful naming convention for naming css classes.

**Review by sanjitsarkar — APPROVED** [review]

File: component/input/input.html (hunk 2 above)
Comment (Sahil25061999) [reply component/input/input.html]: will make the necessary changes

File: component/input/input.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
+@import '/index.css';
+
+.textbox {
+  padding: 8px 1rem;
+  border-radius: 0.4rem;
+  border: 3px solid rgba(114, 114, 114, 0.404);
```
Comment (deekshithmd) [inline component/input/input.css:6]: keeping some color variable for border will be good

File: component/input/input.css [hunk 4]
```diff
@@ -0,0 +1,24 @@
+@import '/index.css';
+
+.textbox {
+  padding: 8px 1rem;
+  border-radius: 0.4rem;
```
Comment (deekshithmd) [inline component/input/input.css:5]: use px for border radius

**Review by deekshithmd — APPROVED** [review]

**Pull request merged**